apps/storybook-react-native/src/index.ts

```diff
@@ -0,0 +1,9 @@
+/**
```

Comment:
non-blocking: Can be removed as this wont be a published package

Reply:
can you clarify?

Reply:
Will this `index.ts` file be required? It's removed from the design-system-react-native package so I don't think we will need it but it can be removed later? https://github.com/MetaMask/metamask-design-system/pull/50/files#diff-3f0cc379008ecb010632c3a6480783ce754def4f4329fc0a178cd38c48eebc90

Reply:
it'll probably be updated so wont be removed

Reply:
Looks like it was removed but maybe you mean it was changed to `index.js`?

<img width="1774" alt="Screenshot 2024-10-17 at 1 26 24 PM" src="https://github.com/user-attachments/assets/c5f67af4-b4d3-4129-886f-5126e88ed367">